docs/docker.md

```diff
@@ -1,38 +1,38 @@
-## Guidelines for GE Dockerization (Containerization)
+## Requirements for GE Dockerization (Containerization)
 
-This page summarizes Guidelines for providing Docker Containers for FIWARE GEs.
-[Click here](https://docs.docker.com/engine/understanding-docker/) for an
+This page summarizes Requirements for providing Docker Containers for FIWARE
+GEs. [Click here](https://docs.docker.com/engine/understanding-docker/) for an
 introduction to Docker.
 
-There are three kinds of guidelines included:
+There are three kinds of requirements included:
```

Comment:
I think the definition of MAY SHOULD MUST is found elsewhere in the document. If so I'd suggest removing this section (i.e. lines 7-14)  to avoid duplication.

Reply:
yep